src/evagram/website/backend/api/test_views.py

```diff
@@ -1,186 +1,113 @@
 from django.test import TestCase
-from api.models import Owners, Plots
+from api.models import *
```

Comment:
Could you also just write `import api.models`? Just a question

Reply:
I didn't want to write out `api.models` every time I am using the Django models.